pegasus/forms/volunteer_engineer_submission_2015.rb

```diff
@@ -108,9 +108,15 @@ def self.process(data)
   end
 
   def self.solr_query(params)
-    # Remove and add UNSUBSCRIBE_HOC before and after each Hour of Code.
     query = "kind_s:\"#{name}\" && allow_contact_b:true && -unsubscribed_s:\"#{UNSUBSCRIBE_FOREVER}\""
 
+    # UNSUBSCRIBE_HOC means a volunteer said "I want to unsubscribe until the next Hour of Code".
+    # We don't want them to be getting volunteer requests until then.  So, if we're not currently
+    # in Hour of Code, don't show that volunteer, and do that by including UNSUBSCRIBE_HOC here.
```

Comment:
Don't we want to open up requests slightly before CS Education Week? Do we need "pre-hoc" and "actual-hoc" modes?